### ISSUE TITLE
Test GH Action Workflow

### ISSUE THREAD

**Issue by daemon1024** [daemon1024/leaflet-environmental-layers#6]

**Comment by daemon1024** [comment]
@github-actions run

```json
{
    "name": "Test-Workflow,
    "description": "Spreadsheet test data",
    "url": "https://docs.google.com/spreadsheets/d/1AR2KRuvxgruqLSCzJoIWxcyLDfPAE3tCifQthTHhpFo/"
}
```

**Comment by daemon1024** [comment]
@github-actions run

```json
{
    "name": "Test-Workflow",
    "description": "Spreadsheet test data",
    "url": "https://docs.google.com/spreadsheets/d/1AR2KRuvxgruqLSCzJoIWxcyLDfPAE3tCifQthTHhpFo/"
}
```

**Comment by jywarren** [comment]
Awesome, so you triggered it with a comment as a test? And it generated https://github.com/daemon1024/leaflet-environmental-layers/pull/7/files?

**Comment by jywarren** [comment]
Looks awesome!

**Comment by daemon1024** [comment]
Yes. The key is we can provide a JSON inside a code block and the GitHub action will generate a layer based on that. 

We will need to trigger this comment from the web page or just ask the volunteer to paste the comment in an issue. And that would complete the cycle. 

Any inputs from your side for this?

Before I work on the web page, I will work on the things you mentioned in https://github.com/publiclab/leaflet-environmental-layers/pull/531

**Comment by jywarren** [comment]
Can we use an issue template maybe? I'm trying to think how to make the formatting really clear. 

This is looking awesome!

**Comment by jywarren** [comment]
Ooh I can imagine a few other refinements as well, like what if it the pr links back to this issue?

**Comment by daemon1024** [comment]
> Can we use an issue template maybe? 

Definitely. Will do.

> what if it the pr links back to this issue?

Ohh it was supposed to, I didn't noticed ig I messed up the link. Will fix this.

**Comment by daemon1024** [comment]
Did some refinement! Can check them out at https://github.com/daemon1024/leaflet-environmental-layers/issues/10

cc @jywarren
